### PR TITLE
Hot fix for chatbot related interface UI

### PR DESCRIPTION
**Pull request by wskksw**
# Description

Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context. 
List any dependencies that are required for this change.

Closes # (issue number)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This requires a run of `yarn install`

# How Has This Been Tested?

Please describe how you tested this PR (both manually and with tests)
Provide instructions so we can reproduce. 

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code where needed 
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any UI changes have been checked to work on desktop, tablet, and mobile

### CODE REVIEW

File: packages/app/components/Today/ChatbotToday.tsx [hunk 1]
```diff
@@ -172,7 +171,7 @@ export const ChatbotToday: React.FC = () => {
       <ChatbotContainer>
         <Card
           title="Course chatbot"
-          className=" flex h-[85vh] w-full flex-col overflow-y-auto sm:h-[90vh]"
+          className=" flex h-full max-h-[750px] w-full flex-col overflow-y-auto"
```
Comment (AdamFipke) [inline packages/app/components/Today/ChatbotToday.tsx:174]: yo um did you mean to do this? i think this is going to re-introduce a bug where the chatbot window height is going to be based on the number of queues

File: packages/app/components/Today/ChatbotToday.tsx [hunk 2]
```diff
@@ -14,18 +14,17 @@ import { Feedback } from '../Chatbot/components/Feedback'
...
-  display: flex;
   overflow: hidden;
 `
 const StyledInput = styled(Input)`
   width: 100%;
   margin-top: 0;
 
   @media (min-width: 650px) {
-    width: 80%;
+    width: 100%;
```
Comment (AdamFipke) [inline packages/app/components/Today/ChatbotToday.tsx:27]: also did you mean to do this?

File: packages/app/components/Today/ChatbotToday.tsx [hunk 3]
```diff
@@ -14,18 +14,17 @@ import { Feedback } from '../Chatbot/components/Feedback'
 const ChatbotContainer = styled.div`
   width: 100%;
   @media (min-width: 650px) {
-    width: 80%;
-    height: 80%;
+    width: 90%;
+    height: 100%;
```
Comment (AdamFipke) [inline packages/app/components/Today/ChatbotToday.tsx:18]: and this?

**Comment by wskksw** [comment]
@AdamFipke You are right on the vh one for main chatbot. I am reversing in chatbot question delete/add pull request. I was trying to fix this bug with chatbotToday where input would be covered if the chatbot overflowed.

**Pull request merged**